pkg/trace/api/otlp.go

```diff
@@ -359,14 +359,7 @@ func (o *OTLPReceiver) ReceiveResourceSpans(ctx context.Context, rspans ptrace.R
 	}
 	select {
 	case o.out <- &p:
-		// Stats will be computed for p. Mark the original resource spans to ensure that they don't
-		// get computed twice in case these spans pass through here again.
-		//
-		// Spans can pass through here multiple times because this code path gets used by the:
-		//  - OpenTelemetry Collector Datadog processor, when computing stats.
-		//  - OpenTelemetry Collector Datadog exporter, when flushing.
-		//  - Datadog Agent OTLP Ingest, when used in conjunction with an SDK or a Collector.
-		rspans.Resource().Attributes().PutBool(keyStatsComputed, true)
+		// Do nothing
```

Comment:
```suggestion
		// success
```
Otherwise it sounds like a path where no action was taken.